tensorhive/controllers/reservation.py

```diff
@@ -170,5 +168,9 @@ def delete(id: ReservationId) -> Tuple[Content, HttpStatusCode]:
         return content, status
 
 
-def __is_admin_or_reservation_owner(reservation: Reservation) -> bool:
-    return 'admin' in get_jwt_claims()['roles'] or reservation.user_id == get_jwt_identity()
+def __is_admin() -> bool:
+    return 'admin' in get_jwt_claims()['roles']
```

Comment:
We have the same function in tensorhive.controllers.task. We can move it to utils.

Reply:
Right! I tried to move both is_admin functions to tensorhive/authorization.py and import from there, but then get_jwt_claims() wouldn't work, so I decided to leave identical, redundant is_admin functions in both controllers :(